src/modules/launcher/Wox/ViewModel/ContextMenuItemViewModel.cs

```diff
@@ -10,10 +10,72 @@ namespace Wox.ViewModel
     public class ContextMenuItemViewModel : BaseModel
     {
         public string PluginName { get; set; }
-        public string Title { get; set; }
-        public string Glyph { get; set; }
-        public string FontFamily { get; set; }
-        public ICommand Command { get; set; }
+
+        private string title;
+        private string glyph;
+        private string fontfamily;
+        private ICommand command;
+
+        public string Title {
+            get
+            {
+                return this.title;
+            }
+
+            set
+            {
+                if (value != this.title)
+                {
+                    this.title = value;
+                    OnPropertyChanged("Title");

```

Comment:
Please use 
`OnPropertyChanged(nameof(Title));` to avoid hardcoding strings.

The other option would be to add this into `BaseModel`

```
protected virtual void OnPropertyChanged([CallerMemberName] string propertyName = null)
        {
            PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(propertyName));
        }
```
And here you would only call OnPropertyChanged();

Reply:
Will incorporate it. Thank you.